### PR TITLE
Codemod Update File

### PR DESCRIPTION
**Pull request by waigani**
Fix bug that prevented codemod from updating files.
Fix formatting issue.

### CODE REVIEW

File: app/commands/codemod/writer.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package codemod
```
Comment (clairscent) [inline app/commands/codemod/writer.go]: should have a package comment, unless it's in another file for this package

File: app/commands/codemod/review.go [hunk 2]
```diff
@@ -208,11 +208,15 @@ l:
 
 			if cfm.Confirm(0, iss, confirmSRC) {
 
+				issuePos := iss.Position
+				startPos := issuePos.GetStart()
+
 				// TODO(waigani) offsets need to be based off codemod decorator, not review.
```
Comment (clairscent) [inline app/commands/codemod/review.go]: Make sure you add this TODO as an issue, milestone, or in your project management tool; eg. Trello.

File: app/commands/codemod/writer.go (hunk 1 above)
Comment (emersonwood) [reply app/commands/codemod/writer.go]: /ignore

Comment (emersonwood) [reply app/commands/codemod/writer.go]: /ignore remove

**Pull request merged**